rest_framework/schemas/openapi.py

```diff
@@ -556,7 +562,8 @@ def _map_field_validators(self, field, schema):
                 schema['maximum'] = v.limit_value
             elif isinstance(v, MinValueValidator):
                 schema['minimum'] = v.limit_value
```

Comment:
Also noticed that Decimal type is not being handled for MaxValueValidator and MinValueValidator currently. I guess it should check whether limit value is Decimal and then also whether COERCE_DECIMAL_TO_STRING is set in order to determine whether to cast to float or just ignore?

Reply:
perhaps something like: 
```
            elif isinstance(v, MaxValueValidator) and (not isinstance(v.limit_value, Decimal) 
                                                       or not api_settings.COERCE_DECIMAL_TO_STRING):
                    schema['maximum'] = v.limit_value
            elif isinstance(v, MinValueValidator) and (not isinstance(v.limit_value, Decimal) 
                                                       or not api_settings.COERCE_DECIMAL_TO_STRING):
                schema['minimum'] = v.limit_value
```

Reply:
@kimbriancanavan That looks like it could be pulled into a separate PR with a small unit test (just on map_field())

Reply:
I agree with making that a separate PR. My primary focus here is ensuring the `string` type is used appropriately given I work with currency amounts.